src/services/NodeMonitor.ts

```diff
@@ -206,6 +206,7 @@ export class NodeMonitor {
 				nodeWithInfo = {
 					...nodeWithInfo,
 					...nodeInfo,
+					host: nodeInfo?.host || nodeWithInfo.host,
```

Comment:
This is probably fine, but I would use the host info from ``node/peers. ``.
Just FYI, I can put ``127.0.0.1`` in the node config.